### ISSUE TITLE
Indicate corrected fields on UI

### ISSUE THREAD

**Issue by dvenprasad** [AlexsLemonade/refinebio-frontend#215]
Some of the fields are corrected (https://github.com/AlexsLemonade/refinebio/issues/165) but on the front-end it doesn't indicate which fields are corrected. 

From https://github.com/AlexsLemonade/refinebio/issues/246, this solution was proposed:
<img width="976" alt="40072698-ad1ee3bc-5842-11e8-9028-5e4aa34ea342" src="https://user-images.githubusercontent.com/15315514/43589748-606ce3dc-963d-11e8-9686-1ecd9b642b17.png">

An alternate solution, as Stef proposed would be to indicate corrected fields with `*`and have a footnote at the end of the table and a link to docs. 

I would prefer the first solution but I'm not sure how customizable the tables we're using is.  

@Willv19 What do you think?

**Comment by Willv19** [comment]
Are the type-ahead filters a separate thing, or are they part of this issue?

**Comment by dvenprasad** [comment]
Type-aheads aren't part of this issue.

**Comment by jaclyn-taroni** [comment]
Notes from tech team meeting
* Expose source metadata (e.g., data that is not getting harmonized) 
* Note indicating some fields get harmonized -> link to docs

**Comment by arielsvn** [comment]
> I would prefer the first solution but I'm not sure how customizable the tables we're using is.

I like the first option as well, we shouldn't have any problems with the table.

I have one question, from where are we going to get the original values of the corrected fields? It seems this isn't being returned by the `/samples` endpoint. @Miserlou @kurtwheeler

**Comment by kurtwheeler** [comment]
That's correct and we don't really have any way of doing so because there's not an easy mapping between the two. Those corrected fields are the result of running this: https://github.com/AlexsLemonade/refinebio/blob/dev/foreman/data_refinery_foreman/surveyor/harmony.py, which doesn't lend itself to being undone on the fly.

This is what Jackie's comment is about. Since we cannot show this mapping, what we've decided to do is make a note indicating some fields get harmonized and have it link to the docs about that and then also make it so the user can display the source metadata (which IS returned by the `/samples` endpoint).   Between the two the user should have enough information to make sense of the metadata.

**Comment by jaclyn-taroni** [comment]
I was typing something similar when @kurtwheeler beat me to it! I will add that we are able to/currently do give folks the source values in the form of annotations (see https://github.com/AlexsLemonade/refinebio/pull/391/commits/2420cb59eee2c16a08118e1c33f63f5becde4bac) when they download data. If we'd like to do something in addition to that, we don't have a design for it. Tagging @dvenprasad.

**Comment by dvenprasad** [comment]
![screen shot 2018-08-09 at 4 31 39 pm](https://user-images.githubusercontent.com/15315514/43924418-97f02796-9bf2-11e8-8711-09cf95554164.png)


![screen shot 2018-08-09 at 4 31 47 pm](https://user-images.githubusercontent.com/15315514/43924428-9ca3759a-9bf2-11e8-8189-7f58a452e8bc.png)


Invision: [https://invis.io/9SNAJ70ME4D#/313673394_experiment-Page-Access-Metadata](https://invis.io/9SNAJ70ME4D#/313673394_experiment-Page-Access-Metadata)

@arielsvn let me know if you need anything else.

**Comment by Miserlou** [comment]
Source Metadata is on a per-sample basis. For experiments with hundreds
with requests, that's not going to be very useful. I like the dialog but
think that it should clicked from inside the last table cell.

**Comment by dvenprasad** [comment]
Ah okay. I thought all the sample metadata were already together. 
Yeah in that case, I agree. We should add a column at the end and link out.

**Comment by jaclyn-taroni** [comment]
Related: AlexsLemonade/refinebio#515
